Core/AppRuntime/Source/AppRuntimeWin32.cpp

```diff
@@ -7,12 +7,20 @@
 
 namespace Babylon
 {
+    namespace
+    {
+        constexpr size_t FILENAME_BUFFER_SIZE = 1024;
+    }
+
     void AppRuntime::RunPlatformTier()
     {
         HRESULT hr = CoInitializeEx(nullptr, COINIT_APARTMENTTHREADED);
         assert(SUCCEEDED(hr));
         auto coInitScopeGuard = gsl::finally([] { CoUninitialize(); });
 
-        RunEnvironmentTier();
+        char filename[FILENAME_BUFFER_SIZE];
```

Comment:
do we need to do this for uwp?

Reply:
It seems not, presumably because of the way UWP apps are run differently in the debugger versus Win32 apps.

Reply:
It could be because it's sandboxed.